transmartApp/grails-app/init/org/transmartproject/app/BootStrap.groovy

```diff
@@ -50,11 +50,6 @@ class BootStrap {
 
         // force marshaller registrar initialization
         grailsApplication.mainContext.getBean 'marshallerRegistrarService'
-
-        if ('clientCredentialsAuthenticationProvider' in
-                grailsApplication.config.grails.plugin.springsecurity.providerNames) {
-            OAuth2SyncService.syncOAuth2Clients()
-        }
     }
 
```

Comment:
And tm-server Bootstrap.groovy and tm-oauth Bootstrap.groovy are exact copies. That also needs deduplication. I think tm-oauth doesn't need most of what's in there.